src/package-manager.ts

```diff
@@ -52,13 +52,13 @@ export const packageManagers: PackageManager[] = [
   {
     name: "yarn",
     command: "yarn",
-    majorVersion: "1.0.0",
+    majorVersion: "1",
     lockFile: "yarn.lock",
   },
   {
     name: "yarn",
     command: "yarn",
-    majorVersion: "3.0.0",
+    majorVersion: "3",
```

Comment:
It would probably be a breaking change, but from a usability standpoint this here would be better:

```ts
name: yarn-berry
majorVersion: ["3", "4"]
```

Because right now on any new yarn berry project (the current major is 4) this whole detection system won't work. And there is no real differentiation between yarn classic and yarn berry.

So in my own code I need to write this at the moment:

```ts
if (pm.name === 'yarn' && (pm.majorVersion === "3" || pm.majorVersion === "4")) {}
```

Would be cool to be able to write this instead:

```ts
if (pm.name === 'yarn-berry') {}
```